### PR TITLE
update wef field to set the work item on the board column

### PR DESCRIPTION
**Pull request by danhellem**
### Sample snippet contributions

For client library sample snippets, review and follow the [snippet contribution guidance](https://github.com/Microsoft/vsts-dotnet-samples/blob/master/ClientLibrary/Snippets/contribute.md)

* [x] Follow organization guidelines
* [x] Use real types (avoids `var`)
* [x] Use `Context.Log` (avoids Console.WriteLine)
* [x] Sample should clean up resources they create
* [x] Avoid samples that arbitrarily destroy data

### CODE REVIEW

**Comment by danhellem** [comment]
@willsmythe and/or @wnjenkin please approve PR

File: ClientLibrary/Samples/Work/BoardsSample.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            WorkHttpClient workClient = connection.GetClient<WorkHttpClient>();
+
+            TeamProjectReference project = ClientSampleHelpers.FindAnyProject(this.Context);
+            WebApiTeamRef team = ClientSampleHelpers.FindAnyTeam(this.Context, project.Id);
+
+            TeamContext context = new TeamContext(project.Id, team.Id);
+            context.Team = team.Name;
+            context.Project = project.Name;          
+
+            Board board = workClient.GetBoardAsync(context, "Stories").Result;
```
Comment (willsmythe) [inline ClientLibrary/Samples/Work/BoardsSample.cs]: Is `Stories` a special keyword? If so, is there a constant variable (in the public client) that can be referenced here instead?

Comment (danhellem) [reply ClientLibrary/Samples/Work/BoardsSample.cs]: "Stories" is just a work item type and can change depending on the process. You can make it a constant but that won't really help :(

File: ClientLibrary/Samples/WorkItemTracking/WorkItemsSample.cs [hunk 2]
```diff
@@ -1101,6 +1101,56 @@ public WorkItemTemplate ReplaceTemplate()
...
+                 {
+                     Operation = Operation.Add,
+                     Path = "/fields/" + wefField,
+                     Value = targetColumn
+                 }
+            );
+
+            var result = witClient.UpdateWorkItemAsync(patchDocument, Convert.ToInt32(workItem.Id)).Result;
+
+            return "";
```
[the remark this answers is not in the document]
Comment (danhellem) [reply ClientLibrary/Samples/WorkItemTracking/WorkItemsSample.cs]: okay...that is fair. good catch. I missed that

[the remark this answers is not in the document]
Comment (danhellem) [reply ClientLibrary/Samples/WorkItemTracking/WorkItemsSample.cs]: fixed

File: ClientLibrary/Samples/Work/BoardsSample.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+            Console.WriteLine("Columns for 'Stories' Board for Project '{0}' and Team '{1}'", context.Project, context.Team);
```
[the remark this answers is not in the document]
Comment (danhellem) [reply ClientLibrary/Samples/Work/BoardsSample.cs]: Fixed. Will go back and clean up old code later

**Pull request merged**